lib/pages/admin/delete_choice_board_category.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+/// Deletes a choiceboard category given ID
+class DeleteChoiceBoardCategory extends StatelessWidget {
+  final String categoryId;
+
+  const DeleteChoiceBoardCategory({Key? key, required this.categoryId})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // Return to admin screen if user cancels choice
+    Widget cancelButton = TextButton(
+      child: Text("No"),
+      onPressed: () {
+        Navigator.pop(context);
+      },
+    );
+
+    // Delete document from firebase collection
+    void deleteFromCollection(String collectionName) {
```

Comment:
all of these methods (in this but also other files should be placed in a separate file that handles all method related to dealing with categories on the database